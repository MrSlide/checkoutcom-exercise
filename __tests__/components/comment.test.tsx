import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'

import Comment from '../../src/components/comment'

describe('Comment component', function () {
  const testProps = {
    author: 'John Doe',
    comment: 'Lorem ipsum dolore sit amet.',
  }

  test('renders correctly', function () {
    const renderer = new ShallowRenderer()

    const result = renderer.render(
      <Comment {...testProps}/>
    )

    expect(result).toMatchSnapshot()
  })
})
