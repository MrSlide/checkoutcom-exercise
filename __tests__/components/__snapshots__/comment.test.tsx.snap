// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment component renders correctly 1`] = `
<div
  role="comment"
>
  <ItemTitle>
    John Doe
  </ItemTitle>
  <Paragraph>
    Lorem ipsum dolore sit amet.
  </Paragraph>
</div>
`;
